### PR TITLE
Cleanup of routes, and addition of GET /:id for articles

### PR DESCRIPTION
**Pull request by kirinrastogi**
## Description
+ Removed ```GET /``` of keywords
+ Added ```GET /:id``` of articles
+ Cleaned up pulling of articles

## TODO
- Sanitize: #9

### CODE REVIEW

**Comment by kirinrastogi** [comment]
@JackyChiu I fixed the error response in my previous pr
```error: error, -> error,```

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "react-dom": "^15.4.0",
     "react-google-charts": "^1.2.1",
     "request": "^2.79.0",
+    "request-promise": "^4.1.1",
```
Comment (JackyChiu) [inline package.json]: 😍

File: routes/article.js [hunk 2]
```diff
@@ -65,34 +65,31 @@ function dropArticles() {
...
-                        art['keywords'] = getPhrases(art)
-                        art['votes'] = 0
+                Article.findOne({url: old.url})
+                    .then((article) => {
+                        // already in database, do nothing
+                        if (!article) throw new Error('Article not found')
+                    })
+                    .catch((err) => {
+                        // not in db
+                        let artObj = Object.assign({
```
Comment (JackyChiu) [inline routes/article.js]: Why create a new object? What about added properties to the old one and then `new Article(old)`?

File: routes/article.js [hunk 3]
```diff
@@ -111,51 +108,65 @@ router.post('/vote', (req, res) => {
...
 
-    })
-    .catch((error) => {
-        res.status(500).send({
-            error: error,
-            reqBody: req.body
+router.get('/:id', (req, res) => {
+    Article.findById(req.params.id)
+        .then((art) => {
+            res.status(200).send(art)
```
Comment (JackyChiu) [inline routes/article.js]: `res.send(art)` already has a status of 200, no need to explicitly state it

**Comment by JackyChiu** [comment]
@kirinrastogi You can leave it merged, just have a question

**Pull request merged**